Blish HUD/GameServices/ArcDpsService.cs

```diff
@@ -27,6 +33,11 @@ public class ArcDpsService : GameService {
         /// </summary>
         public bool RenderPresent { get; private set; }
 
+        /// <summary>
+        ///     Indicates if the socket listener for the arcdps service is running and arcdps send an update in the last second.
```

Comment:
Its calculated on the setter of HudIsActive which should be updated every frame in game. If that does not happen, RenderPresent is set to false after 1 sec. (In Update Loop)

Reply:
This for example does cover a few edge cases where we are connected socket wise but arc does not actually send anything because some pattern can't be found after an update which disabled the arc api.
In this case the Service is effectively not running even though we are connected.

Reply:
Ahh!  Gotcha.  Thank you!